File: 561-array-partition/561-array-partition.java
class Solution {
    public int arrayPairSum(int[] nums) {
        
        // sort the array
        // track total
        // loop every other iteration add to total
        Arrays.sort(nums);
        int total = 0;
        
        for (int i = 0; i < nums.length; i+=2) {
            total = total + nums[i];
        }
        return total;
    }
}

File: 561-array-partition/561-array-partition.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var arrayPairSum = function(nums) {
    
  
    nums.sort(function(a,b) { return a - b; });
    let total = 0;
    
    for (let i = 0; i < nums.length; i+=2) {
        total = total + nums[i];
    }
    return total;
};